### PR TITLE
Rename volumefrom

### PR DESCRIPTION
**Pull request by antho1404**
depending on https://github.com/mesg-foundation/core/pull/860
fixes https://github.com/mesg-foundation/core/issues/865

Renaming `volumefrom` into `volumeFrom` in the protobuf definition

### CODE REVIEW

File: protobuf/definition/service.proto [hunk 1]
```diff
@@ -9,6 +9,7 @@ message Service {
   string sid = 12;                          // Service's sid.
   string name = 1;                          // Service's name.
   string description = 2;                   // Service's description.
+  Configuration configuration = 8;          // Configurations related to the service
```
Comment (krhubert) [inline protobuf/definition/service.proto]: This is not related to this PR - it should be done in different PR.

Comment (antho1404) [reply protobuf/definition/service.proto]: that's the case, look at the description of the PR, depending on #860
